src/constants_and_utils/messageBuilder.js

```diff
@@ -52,7 +56,7 @@ function getDefaultIsStillVisibleMessage(htmlElement) {
   }.`;
 }
 
-function getDeafultTextTextIsStillPresentOnElementMessage(htmlElement, text) {
+function getDefaultTextTextIsStillPresentOnElementMessage(htmlElement, text) {
```

Comment:
It seems that there's another typo here 😜 

```suggestion
function getDefaultTextIsStillPresentOnElementMessage(htmlElement, text) {
```

Reply:
done